parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -43,6 +44,14 @@ pub type HeaderUpdateOf<T> = HeaderUpdate<
 	<T as Config>::MaxProofBranchSize,
 	<T as Config>::MaxSyncCommitteeSize,
 >;
+pub type HeaderUpdateOfCapella<T> = HeaderUpdateCapella<
```

Comment:
The `ThingOf<T>` pattern is always like this:

```suggestion
pub type HeaderUpdateCapellaOf<T> = HeaderUpdateCapella<
```

Because `Of` refers to the generic parameter `T`